tools/relay/Makefile

```diff
@@ -30,7 +30,6 @@ relay:
 	go build -v -o ./server github.com/vhive-serverless/vSwarm/tools/relay
 
 relay-docker:
-	go mod tidy
```

Comment:
Is it removed by mistake?

Reply:
No its not. `go mod tidy` will be called during the build again inside the container.
This one has no effect and therefore I removed it